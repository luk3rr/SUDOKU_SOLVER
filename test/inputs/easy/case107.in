710000950 040790010 000008007 080030079 009002104 000400020 006000590 001050800 500021040 
