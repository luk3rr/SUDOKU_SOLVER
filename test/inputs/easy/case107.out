Solving the following grid:
7 1 0 | 0 0 0 | 9 5 0 
0 4 0 | 7 9 0 | 0 1 0 
0 0 0 | 0 0 8 | 0 0 7 
------+-------+------
0 8 0 | 0 3 0 | 0 7 9 
0 0 9 | 0 0 2 | 1 0 4 
0 0 0 | 4 0 0 | 0 2 0 
------+-------+------
0 0 6 | 0 0 0 | 5 9 0 
0 0 1 | 0 5 0 | 8 0 0 
5 0 0 | 0 2 1 | 0 4 0 


Solution found :')

7 1 8 | 2 6 4 | 9 5 3 
6 4 5 | 7 9 3 | 2 1 8 
9 3 2 | 5 1 8 | 4 6 7 
------+-------+------
2 8 4 | 1 3 5 | 6 7 9 
3 5 9 | 6 7 2 | 1 8 4 
1 6 7 | 4 8 9 | 3 2 5 
------+-------+------
8 2 6 | 3 4 7 | 5 9 1 
4 7 1 | 9 5 6 | 8 3 2 
5 9 3 | 8 2 1 | 7 4 6 

Algorithm: BFS
Total time: 25 ms
Total expanded states: 3655
